### PR TITLE
Complete AboutUs responsive design

### PR DESCRIPTION
**Pull request by pcmueller**
## Type of Change
- [ ] JSX
- [x] HTML
- [ ] JavaScript
- [x] CSS
- [ ] React
- [ ] React Router
- [ ] Cypress
- [ ] Apollo
- [ ] GraphQL
- [ ] Twilio API
- [x] Refactor/Fix Bugs

## Please describe the changes made
- Completed AboutUs CSS/responsive design, including light/dark mode
- Got our Lighthouse accessibility score back up to 97 - only flags are for color contrast of h1 / form input text / button text, which look pretty good to me, not sure if we can do much more there - I think it's just b/c we have a thinner font w/o a bold option.  

## How has this been tested?
- [ ] In Browser
- [x] In Console
- [ ] Cypress
- [ ] Postman

### CODE REVIEW

**Review by carolineshowscode — APPROVED** [review]
Thanks for correcting the dimensions! Nice job!

File: src/Components/AboutUs/_aboutus.scss [hunk 1]
```diff
@@ -7,37 +7,40 @@
   width: 100%;
 }
 
+.about-us-title {
+  font-size: 2em;
+  text-shadow: #6E6355FF;
+  font-family: $julius;
+  margin: 0.5em auto 0.25em auto;
+}
```
Comment (bhohnco) [inline src/Components/AboutUs/_aboutus.scss:15]: Nice add here Pete!

File: src/Components/AboutUs/_aboutus.scss [hunk 2]
```diff
@@ -165,6 +167,34 @@
...
+    font-size: 16px;
+  }
+
+  .team-grid {
+    width: 83%;
+    height: 38%;
+  }
+
+  .row-title {
+    font-size: 18px;
```
Comment (bhohnco) [inline src/Components/AboutUs/_aboutus.scss:186]: Looks great!

File: src/Components/Form/Form.js [hunk 3]
```diff
@@ -132,6 +133,7 @@ function Form({ contacts, handleEtaChange, userInfo, setContact, setTripIsActive
         onKeyPress={(e) => { e.key === 'Enter' && e.preventDefault(); }}
         options={formattedContacts}
         styles={customStyles}
+        aria-label='select contact dropdown menu'
```
Comment (bhohnco) [inline src/Components/Form/Form.js:136]: Good call on the Aria label here

**Review by bhohnco — APPROVED** [review]

**Pull request merged**